openaev-front/src/admin/components/common/toolBar/BulkToolBar.tsx

```diff
@@ -0,0 +1,97 @@
+import { ClearOutlined, InfoOutlined } from '@mui/icons-material';
+import { AppBar, IconButton, Toolbar, Typography } from '@mui/material';
+import { useTheme } from '@mui/material/styles';
+import { type FunctionComponent } from 'react';
+import { makeStyles } from 'tss-react/mui';
+
+import { useFormatter } from '../../../../components/i18n';
+import { type ToolTasks } from './BulkToolBar-model';
+
+const useStyles = makeStyles()(theme => ({
+  title: {
+    flex: '1',
+    fontSize: '12px',
+  },
+  numberOfSelectedElements: {
+    padding: '2px 5px 2px 5px',
```

Comment:
todo: useTheme as much as possible. And try to follow spacing convention -> 0.5, 1, 2, ect